tests/test_pyxis_gql.py

```diff
@@ -321,6 +321,97 @@ def test_pyxis_graphql_find_images_by_nvr():
+                    "content_sets": ["rhel-8-for-x86_64-baseos-rpms"],
+                    "edges": {
+                        "rpm_manifest": {
+                            "data": {
+                                "rpms": [
+                                    {
+                                        "name": "foo",
+                                        "nvra": "foo-10-123.el8.noarch",
+                                        "srpm_name": "foo",
+                                        "srpm_nevra": "foo-10-123.el8.src",
+                                    }
+                                ]
+                            }
+                        }
+                    },
+                    "parent_brew_build": "ubi8-minimal-container-8.6-100.1582220001",
+                    "parsed_data": {},
+                    "repositories": [
+                        {
+                            "published": True,
+                            "registry": "registry.example.com",
+                            "repository": "dummy/foobar-rhel8",
+                            "tags": [{"name": "v0.13"}],
+                        }
+                    ],
+                },
+                {
+                    "architecture": "arm64",
+                    "brew": {"build": "foobar-container-v0.13.0-12.1582340001"},
+                    "content_sets": ["rhel-8-for-aarch64-baseos-rpms"],
+                    "edges": {
+                        "rpm_manifest": {
+                            "data": {
+                                "rpms": [
+                                    {
+                                        "name": "foo",
+                                        "nvra": "foo-10-123.el8.noarch",
+                                        "srpm_name": "foo",
+                                        "srpm_nevra": "foo-10-123.el8.src",
+                                    }
+                                ]
+                            }
+                        }
+                    },
+                    "parent_brew_build": "ubi8-minimal-container-8.6-100.1582220001",
+                    "parsed_data": {},
+                    "repositories": [
+                        {
+                            "published": True,
+                            "registry": "registry.example.com",
+                            "repository": "dummy/foobar-rhel8",
+                            "tags": [{"name": "v0.13"}],
+                        }
+                    ],
+                },
+            ],
+            "error": None,
+            "page": 0,
+            "page_size": 50,
+            "total": 2,
+        }
+    }
+
+    pyxis_gql = PyxisGQL(url="graphql.pyxis.local", cert=("/path/to/crt", "/path/to/key"))
+    flexmock(Client).should_receive("execute").and_return(copy.deepcopy(result))
+
+    nvrs = ["python-v3.6-201910221723", "bash-4.2.46-34.el7.x86_64"]
```

Comment:
This doesn't make sense, the mocked result only contains NVRs of `["foobar-container-v0.13.0-12.1582340001"]`, you should call it with the reasonable NVRs.

Reply:
changed the unit test to use "foobar-container-v0.13.0-12.1582340001" as discussed